apptentive-react-native.podspec

```diff
@@ -1,19 +1,20 @@
+require "json"
+
+package = JSON.parse(File.read(File.join(__dir__, "package.json")))
+
 Pod::Spec.new do |s|
   s.name         = "apptentive-react-native"
-  s.version      = "5.8.3"
-  s.summary      = "Apptentive SDK module for React Native"
+  s.version      = package["version"]
+  s.summary      = package["description"]
+  s.homepage     = package["homepage"]
+  s.license      = package["license"]
+  s.authors      = package["author"]
+
+  s.platforms    = { :ios => "11.0" }
```

Comment:
Getting 404 for this repo.

Reply:
Also, we should move this to the Apptentive git repository, if it is staying in Harry's private space.

Reply:
Good catch!